src/commands/domains/delete.ts

```diff
@@ -0,0 +1,60 @@
+import { Command, flags } from '@oclif/command'
```

Comment:
I would assume the lint-test would need to get fixed as well.

Reply:
Later

Reply:
Yea non essential at the moment